### PR TITLE
Add language selector

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/navbar/Settings.scss [hunk 1]
```diff
@@ -6,17 +6,43 @@
...
+  height: 40px;
+  display: flex;
+  align-items: center;
+  padding-left: 10px;
+  padding-right: 10px;
+  width: 100%;
+  box-sizing: border-box;
+
+  .MuiSelect-select:focus {
+    background-color: white;
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/navbar/Settings.scss:46]: should be a variable

File: packages/tempus-client_v2/src/components/navbar/Settings.tsx [hunk 2]
```diff
@@ -1,34 +1,32 @@
...
-import getText from '../../localisation/getText';
+import getText, { Language } from '../../localisation/getText';
 import SlippageIcon from '../icons/SlippageIcon';
 import Typography from '../typography/Typography';
 import Spacer from '../spacer/spacer';
 import InfoTooltip from '../infoTooltip/infoTooltip';
 import PercentageInput from '../percentageInput/percentageInput';
 
 import './Settings.scss';
+import LanguageIcon from '../icons/LanguageIcon';
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/navbar/Settings.tsx:13]: please move this import before the `.scss` import

**Review by danieledamiani — DISMISSED** [review]
approved, would be nice to fix a small thing

File: packages/tempus-client_v2/src/components/navbar/Settings.scss (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/navbar/Settings.scss]: fixed

File: packages/tempus-client_v2/src/components/navbar/Settings.tsx (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/navbar/Settings.tsx]: fixed

**Comment by lazarmitic** [comment]
fixed all comments :D

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**